tests/appsec/waf/test_addresses.py

```diff
@@ -12,10 +12,11 @@
 
 
 # WAF/current ruleset don't support looking at keys at all
-@released(golang="?", dotnet="?", java="?", nodejs="?", php="?", python="?", ruby="1.0.0.beta1")
+@released(golang="?", dotnet="?", java="?", nodejs="2.6.0", php="?", python="?", ruby="1.0.0.beta1")
 class Test_UrlQueryKey(BaseTestCase):
     """Appsec supports keys on server.request.query"""
 
+    @irrelevant(reason="no non-encoded rule on query yet")
```

Comment:
I think the only reason this test was working for ruby is because he makes the querystring a list of kv, and the WAF interprets this as a value, not as a key ;)

Reply:
@lloeki feel free to confirm my hypothesis and update the test accordingly

Reply:
Waiting form confirmation from @lloeki then you can remove the test.

Reply:
Confirmed, this should not read releasd for ruby above.